NEWS.md

```diff
@@ -27,6 +27,12 @@
 
 ### What's changed
 
+-   For compatibility with CRAN, the number of threads used by Polars may be automatically set to 2 (#720).
+    To disable this behavior and have the maximum number of threads used automatically,
+    one of the following ways can be used:
+    -   Build the Rust library with the `disable_auto_limit_max_threads` feature.
+    -   Set the `polars.disable_auto_limit_max_threads` option to `TRUE` with the `options()` function
```

Comment:
I find it hard to understand what `disable_auto_limit_max_threads = TRUE` does, can we rename if `use_max_threads`? It's clearer to set `polars.use_max_threads = TRUE`

Reply:
Good point.
I was wondering if it would be worth aligning the name with feature, but it would definitely be easier to understand if the option name was changed.

Reply:
I changed it to `polars.limit_max_threads`.
Given that the environment variable is `POLARS_MAX_THREADS`, I think the term `max` confusing.